openwisp_radius/tasks.py

```diff
@@ -55,6 +56,30 @@ def delete_unverified_users(older_than_days=1, exclude_methods=''):
     )
 
 
+@shared_task
+def unverify_inactive_users():
+    if not app_settings.UNVERIFY_INACTIVE_USERS:
+        return
+    RegisteredUser = load_model('RegisteredUser')
+    RegisteredUser.objects.filter(
+        user__is_staff=False,
+        user__last_login__lt=timezone.now()
+        - timedelta(days=app_settings.UNVERIFY_INACTIVE_USERS),
+    ).update(is_verified=False)
```

Comment:
Please move this logic to a model method and just call the model method here.
Let's stop putting logic in celery tasks, let's consider these tasks as some sort of views.